vault.js

```diff
@@ -1,4 +1,22 @@
 'use strict';
 module.exports = function() {
-  
+    let theVault = {};
+
+    var getValue = function(key){
+        if (theVault.hasOwnProperty(key) === false){
```

Comment:
instead of checking if the output of `.hasOwnProperty` is false, you can use the not operator to make this simpler:
```
  if (!theValue.hasOwnProperty(key) { 
```